gokart/task.py

```diff
@@ -268,3 +270,18 @@ def restore(cls, unique_id):
     @luigi.Task.event_handler(luigi.Event.FAILURE)
     def _log_unique_id(self, exception):
         logger.info(f'FAILURE:\n    task name={type(self).__name__}\n    unique id={self.make_unique_id()}')
+
+    @luigi.Task.event_handler(luigi.Event.START)
+    def _dump_module_versions(self):
+        self.dump(self.get_module_versions(), self._get_module_versions_target())
+
+    def _get_module_versions_target(self):
+        return self.make_target(f'log/module_versions/{type(self).__name__}.txt')
+
+    def get_module_versions(self) -> str:
```

Comment:
Why is this function `get_module_versions ` public? I think `_get_module_versions` is better.